### ISSUE TITLE
Accessing WebUI through PathPrefixStrip frontend returns 404

### ISSUE THREAD

**Issue by davendesai** [containous/traefik#1081]
### What version of Traefik are you using (`traefik version`)?

`Traefik version v1.1.2 built on 2016-12-15`

### What is your environment & configuration (arguments, toml...)?

```yml
traefik:
    container_name: traefik
    image: hypriot/rpi-traefik:latest
    command: --web --docker --docker.domain=rpi.home --logLevel=DEBUG
    restart: always
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - /dev/null:/etc/traefik/traefik.toml
    ports:
      - 80:80
    labels:
      - traefik.frontend.rule=PathPrefixStrip:/traefik
      - traefik.port=8080
```

### What did you do?

Try to access the WebUI via Traefik frontend.

### What did you expect to see?

The have the URL http://rpi.home/traefik take me to the WebUI via appending `/dashboard/` - i.e. http://rpi.home/traefik/dashboard/

### What did you see instead?

Instead of appending the subdirectory `/dashboard/` to the current URL, the following HTML is received:
```
<a href="/dashboard/">Found</a>.
```
This breaks the PathPrefixStrip frontend rule and instead directs to http://rpi.home/dashboard/ which results in a 404.

**Comment by cyrilgdn** [comment]
+1 Same problem here.

**Comment by barhun** [comment]
AFAIK, Traefik Web UI Server doesn't support relative URIs; so you must try something like the following to achieve access without hitting the root path. After Traefik launches, go to _http://yourdomain.tld/dashboard/_ and voila!

```
labels:
  - "traefik.frontend.rule=PathPrefix:/dashboard,/api,/health"
  - "traefik.port=8080"
```

**Comment by timoreimann** [comment]
With #985 being merged, we could now teach the dashboard to look for the header containing the stripped prefix path and build relative URLs accordingly.

**Comment by davendesai** [comment]
@barhun That's a solution that actually completely flew over my head. Might take a look into that in an upcoming weekend 😄

**Comment by ldez** [comment]
Closed by #1638, in the version 1.3.0-rc3.

**Comment by cyrilgdn** [comment]
@ldez Does the fix on PathPrefixStrip really close this issue? As @timoreimann said:
> we could now teach the dashboard to look for the header containing the stripped prefix path and build relative URLs accordingly.

But it's not the case ATM no?

**Comment by ldez** [comment]
I closed "manually" issues related to #1638 because Github does not automatically close during a merge on a branch (eg. not on the master branch).
There may have an error in the list of referenced issues.

Nevertheless, #1233 seems to add a workaround.

**Comment by MrOerni** [comment]
Same problem here.

**Comment by jericdeleon** [comment]
Having difficulty for accessing the WebUI through frontend rules in Docker Swarm as well (Gateway Timeout)

(not just PathPrefix, but with Host rules too)

**Comment by ldez** [comment]
Closed by #2686
